Rakefile

```diff
@@ -50,6 +52,14 @@ task :rdoc do
   Rake::Task['rdoc:rerdoc'].invoke
 end
 
+desc 'rest test examples'
```

Comment:
Is there any particular reason this was implemented as a separate test, rather than part of the existing `test/integration` tests?

Reply:
Adding test as part  of the integration suite.

Added separately for isolated testing, as it requires beef to be running.